btk/draw_blends.py

```diff
@@ -403,7 +417,7 @@ def render_blend(self, blend_catalog, psf, filt, survey, extra_data):
         if self.add_noise:
             if self.verbose:
                 print("Noise added to blend image")
-            generator = galsim.random.BaseDeviate(seed=np.random.randint(99999999))
+            generator = galsim.random.BaseDeviate(seed=self.rng.integers(100000))
```

Comment:
Yes, it will give the same seed to the Galsim one everytime so it should be fine.

Reply:
For clarity, please separate the logic in two lines 

```python
galsim_seed = self.rng.integers(xxx)
galsim.random.BaseDeviate(seed=galsim_seed)
```

But I am still not convinced that this will always return the same seed. It will depend on the order of the computations. If there is a `if` statement somewhere before that calls to `self.rng` this will return a different value for the seed.

That's why I think we need to be careful here because the way you tested it might not be the only way it will be used.

Reply:
Why would there be an additional `if` statement if we use the same parameters for the generation ?